godot-core/src/builtin/rect2.rs

```diff
@@ -44,6 +44,184 @@ impl Rect2 {
         }
     }
 
+    /// Returns a rectangle with equivalent position and area, modified so that the top-left corner is the origin and `width` and `height` are positive.
```

Comment:
This is confusing:
* position can be different, not equivalent (I know what you mean, but in this context it is assumed to refer to the field).
* width/height are not positive, but non-negative

I'd keep it simple, something like:
> Returns a rectangle with the same geometry, with top-left corner as `position` and non-negative size.

Reply:
I just copied the same contents you can find in the official docs (https://docs.godotengine.org/en/stable/classes/class_rect2.html#method-descriptions). Do you think I should really change that?

Reply:
Yes, the official documentation is often confusing or under-specified.

If we can do better, there's no reason to copy bad docs.